redisinsight/ui/src/pages/rdi/pipeline/components/refresh-pipeline-popover/RefreshPipelinePopover.tsx

```diff
@@ -23,12 +27,27 @@ const RefreshPipelinePopover = () => {
 
   const { loading, data } = useSelector(rdiPipelineSelector)
 
+  const { setFieldValue, resetForm } = useFormikContext<IPipeline>()
+
+  const history = useHistory()
   const { rdiInstanceId } = useParams<{ rdiInstanceId: string }>()
+
   const dispatch = useDispatch()
 
   const handleRefreshClick = () => {
     setIsPopoverOpen(false)
-    dispatch(fetchRdiPipeline(rdiInstanceId))
+    dispatch(
+      fetchRdiPipeline(rdiInstanceId, (pipeline: Nullable<IPipeline>) => {
+        resetForm()
+        setFieldValue('config', pipeline?.config)
+        setFieldValue('jobs', pipeline?.jobs)
+
+        // redirect to empty pipeline page if pipeline is empty
```

Comment:
Please correct me if i wrong, but we don't need empty pipeline page. If there is no pipeline we have to show select pipeline mode page. am i wrong?

Reply:
Here it is in the mockups: https://www.figma.com/file/z08rwfM11zwq2Mu5W0qjgJ/RDI?type=design&node-id=478-57664&mode=design&t=87oytK8YEBdPhtLw-0

And the card called it out as one of the requirements.

@ViktarStarastsenka - Did we decide that the empty RDI pipelines page isn't needed?

Reply:
yes, but it will be covered later in a separate story

Reply:
actually, @kyle-marcum , I apologize, but this screen does not align with the new pipeline mode configuration page.
This is my fault, but looks like we do not need this page anymore since it will be covered with the pipeline mode configuration page

Reply:
should I go ahead and remove as a part of this PR or hold off?

Reply:
remove it as a part of this PR please